### ISSUE TITLE
Add VPS Providers to Resources

### ISSUE THREAD

**Issue by sco1** [python-discord/site#177]
Since it comes up fairly often, it would be nice to include a list of some popular VPS providers, along with some basic info.

Info could possibly include:
- Free/paid (base tier cost?)
- Payment types accepted (CC, PayPal, BTC, etc.)
- Services offered (CPU/Memory heavy options, distros, load balancers, container things, monitors, DNS, etc.) 
- Server locations

This should also get a tag that links to the relevant section of the Resources page.

**Comment by jchristgit** [comment]
Honestly, I don't think there's any decent free VPS provider, is there? There's a bunch of rather cheap ones though that we should probably recommend.

**Comment by jos-b** [comment]
Yeah I don't think there are any good VPS providers out there. You will almost always end up signing up to a scam.

**Comment by lemonsaurus** [comment]
I do not think this belongs on resources. I want to keep our resource list concise and strictly about python. I don't see any harm in having a VPS guide in our wiki once the move to django is done, though, if anyone wants to write it. 

Please note that there's no need to wait until the django rewrite is complete before making such a guide, since our wiki on django will just be markdown. You could safely author this for the github wiki or even just as a tag, for now.

**Comment by Den4200** [comment]
I brought this up in `#organisation` and it seems that we want to link to a curated list of good VPS's, rather than creating our own.

**Comment by lemonsaurus** [comment]
Since there's been no movement on this for a very long time and there's no actual implementation suggestion here, I'm gonna close it. If anyone really want to see this happen, open a new issue with a better specification.
